dags/process_elife_zip_dag.py

```diff
@@ -221,6 +221,14 @@ def strip_related_article_tags_from_article_xml(**context) -> bytes:
     return etree.tostring(article_xml, xml_declaration=True, encoding='UTF-8')
 
 
+def strip_object_id_tags_from_article_xml(**context) -> bytes:
```

Comment:
the implementation of this could be extracted into a private function called also from `strip_related_article_tags_from_article_xml`, since it's identical but for the xpath query

Reply:
turns out lxml provides this functionality. Updated.